src/Admin/Actions.php

```diff
@@ -46,7 +46,19 @@ public function register_assets() {
 	}
 
 	public function save_admin_settings() {
-		$post_data = $_POST;
+		//$post_data = $_POST;  
```

Comment:
@Ankur0812 Remove commenting from all the files

Reply:
Done